x-pack/plugin/sql/qa/src/main/java/org/elasticsearch/xpack/sql/qa/jdbc/ResultSetTestCase.java

```diff
@@ -885,10 +894,16 @@ public void testGettingDateWithoutCalendar() throws Exception {
         Long randomLongDate = randomNonNegativeLong();
         indexSimpleDocumentWithTrueValues(randomLongDate);
         
-        doWithQuery(SELECT_ALL_FIELDS, (results) -> {
+        // Create simple timezones (in the form of Z/GMT+/-x so that timezones offsets differences between
+        // Java versions (the one server runs on and the one the test runs on) will not skew the tests' results
```

Comment:
"...between JVMs" (..."
"...the one **the** server..."

Reply:
Nvmd these suggestions, just read the explanation in https://github.com/elastic/elasticsearch/issues/50035#issuecomment-579173566